<launch>
	<master auto="start"/>

	<!-- Run the map server -->
	<node name="map_server" pkg="map_server" type="map_server" args="$(find elektron_navigation)/maps/map_5cm.pgm 0.05"/>

	<!--- Run AMCL -->
	<include file="$(find elektron_navigation)/launch/amcl_diff.launch" />

	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
		<rosparam file="$(find elektron_navigation)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
		<rosparam file="$(find elektron_navigation)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
		<rosparam file="$(find elektron_navigation)/config/local_costmap_params.yaml" command="load" />
		<rosparam file="$(find elektron_navigation)/config/global_costmap_params.yaml" command="load" />
		<rosparam file="$(find elektron_navigation)/config/base_local_planner_params.yaml" command="load" />
		
		<param name="controller_frequency" type="double" value="5"/>
	</node>
</launch>